.travis.yml

```diff
@@ -11,7 +11,6 @@ env:
     
     - CONDA_PY=27
     - CONDA_PY=35
-    - CONDA_PY=36
```

Comment:
For some reason this dropped Python 3.6 on macOS. Fixing with PR ( https://github.com/conda-forge/ipyparallel-feedstock/pull/32 ).